spec/README.md

```diff
@@ -67,6 +67,7 @@ Example structure for the German language file, which only contains lower case c
         "alphabet": "Latn",
         "continent": "EU",
         "language": "German",
+        "variant": "",
```

Comment:
I added it to the spec to show the property. Since it's optional, I didn't bother to update the `de.js` file. So you're saying we should remove it from the example? Maybe we can add a comment saying it isn't necessary if empty.